### DISCUSSION TITLE
Youtube potentially blocking ad blockers

### DISCUSSION THREAD

**Discussion by benjamingwynn** [uBlockOrigin/uAssets#18063] [General] [answered]
YouTube have recently started displaying a banner to some users that disables ad block.

I'm not currently aware if uBlock is able to workaround this banner as I haven't seen it, but if anyone does see the ad-block-block with uBlock Origin they could potentailly share their experience here.

![Youtubes banner blocking ads](https://regmedia.co.uk/2023/05/10/screenshot_ad_block.png)

Article linked for convenience: https://www.theregister.com/2023/05/10/youtube_ad_blockers/

**Comment by MasterKia** [comment, marked as answer]
https://github.com/uBlockOrigin/uAssets/issues/7636
